src/Microsoft.AspNetCore.Hosting/WebHostBuilder.cs

```diff
@@ -29,9 +29,10 @@ public class WebHostBuilder : IWebHostBuilder
         private readonly List<Action<ILoggerFactory>> _configureLoggingDelegates;
 
         private IConfiguration _config;
-        private ILoggerFactory _loggerFactory;
         private WebHostOptions _options;
         private bool _webHostBuilt;
+        private Func<IHostingEnvironment, IConfiguration, ILoggerFactory> _loggerFactoryFunc;
+        private Action<IConfigurationBuilder, IHostingEnvironment> _configureConfigurationBuilderDelegate;
```

Comment:
ConfigureServices and ConfigureLogging allow multiple calls. I understand why creating the logger factory would not, but configuring configuration easily could.